site/content/en/announcements/v1.0.md

```diff
@@ -195,6 +195,6 @@ The release adds a ton of features and functionality. Here are some highlights:
 - Added Support for egctl translate from gateway-api resources to IR
 
 [Release Notes]: https://github.com/envoyproxy/gateway/blob/main/release-notes/v1.0.0.yaml
-[matrix]: https://gateway.envoyproxy.io/v1.0.0/install/matrix/
```

Comment:
we shouldn't rename the folder versions.
I thinks a minor release should share same version doc with major release.
maybe rename `v1.0.0` to `v1.0.x`.

Reply:
This can be discussed later, for how to manage patch release docs, some opts:
1. change path to latest patch release version: like v1.0.1
2. do not contain patch release version in name: like v1.0